capstone/src/main/java/com/google/sps/servlets/clubs/ClubServlet.java

```diff
@@ -82,7 +80,8 @@ public void doPostHelper(
       String clubName = request.getParameter(Constants.PROPERTY_NAME);
       String description = request.getParameter(Constants.DESCRIP_PROP);
       String website = request.getParameter(Constants.WEBSITE_PROP);
-      BlobKey key = getBlobKey(request, Constants.LOGO_PROP, blobstore);
+      BlobstoreUtil blobstoreUtil = new BlobstoreUtil();
```

Comment:
If this class is truly a static utils class (and all the methods are static), you won't need to define this here and you can just import the class and use the methods kind of like with the Constants class.

Reply:
Got it, thank you!